processing/pom.xml

```diff
@@ -75,6 +75,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
can you add dependency into profile hadoop-3.1

Reply:
currently there is profile for hadoop, do you want me to add a profile for it? As it was default i didnt add